platform/posix/transport/CMakeLists.txt

```diff
@@ -52,8 +52,7 @@ target_link_libraries( openssl_posix
                        PUBLIC
                           sockets_posix
                        PRIVATE
-                          OpenSSL::Crypto
-                          OpenSSL::SSL
+                          ${OPENSSL_LIBRARIES}
```

Comment:
Where is `OPENSSL_LIBRARIES` variable defined?

Reply:
It is defined by the built-in `FindOpenSSL.cmake` file once OpenSSL is found:
https://cmake.org/cmake/help/v3.6/module/FindOpenSSL.html

Reply:
Can you add a comment to provide a link to this page?